meta/appveyor.yml

```diff
@@ -0,0 +1,42 @@
+# This CI configuration tests the cosmwasm-template repository itself,
+# not the resulting project. We want to ensure that
+# 1. the template to project generation works
+# 2. the template files are up to date
+#
+# We chose Appveyor for this task as it allows us to use an arbitrary config
+# location. Furthermore it allows us to ship a Circle CI and Travis config
+# generated for the resulting project.
+
+environment:
+  matrix:
+    - TARGET: x86_64-pc-windows-msvc
```

Comment:
Wow. Nothing has been tested on windows until now.
This is good to add to the `go-cosmwasm` bindings (and/or `wasmd`) as well.